index.html

```diff
@@ -3989,22 +3989,20 @@ <h3>JSON-LD</h3>
 
         <p>
 As elaborated upon in Section <a href="#json-processing"></a>, some software
-applications might not perform full JSON-LD processing. In order to increase
-interoperability, <a>conforming document</a> authors are urged to not use
-JSON-LD features that are not easily detected when not performing full
-JSON-LD processing. This guidance includes:
+applications might not perform full JSON-LD processing. To increase
+interoperability, <a>conforming document</a> authors are urged to avoid using
+JSON-LD features that are easily detected only when performing full
+JSON-LD processing. These features include:
         </p>
 
         <ul>
           <li>
-Not using JSON-LD keywords in the `@context` value that modify document
-processing, such as globally setting `@base` and `@language`, in order to
-increase interoperability.
+Use of JSON-LD keywords in the `@context` value that modify document
+processing, such as global settings of `@base` and `@language`
           </li>
           <li>
-Always using the compact form for JSON-LD terms and types and always ensuring
-that those terms and types are defined in application-specific JSON-LD context
-files.
+Use of the compact form for JSON-LD terms and types and defining those
+terms and types in application-specific JSON-LD context files
```

Comment:
This doesn't make sense with the suggestions above (it inverts what should be done since this is to be "avoided", so suggest changing to:

```suggestion
Use of the non-compact form for JSON-LD terms and types. Instead prefer using
compact form and define terms and types in application-specific JSON-LD context files.
```